misaligned.c

```diff
@@ -4,12 +4,17 @@
 int printColorMap() {
     const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
     const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
-    int i = 0, j = 0;
+    int i = 0, j = 0, k = 0;
+    char manualmap[16]={};
     for(i = 0; i < 5; i++) {
         for(j = 0; j < 5; j++) {
-            printf("%d | %s | %s\n", i * 5 + j, majorColor[i], minorColor[i]);
+            k = i*5+j;
+            sprintf(manualmap,"%d | %s | %s\n", k, majorColor[i], minorColor[i]);

```

Comment:
If you separate the sprintf into a separate function to formOneRowOfText, you can test it separately.
Later, if we need to change the output to csv or HTML, we can just replace that function alone